go/vt/vtgate/planbuilder/testdata/ddl_cases.txt

```diff
@@ -293,3 +293,85 @@
     "Query": "drop view a"
   }
 }
+
+# Truncate table with qualifier
+"truncate user.user_extra"
+{
+  "QueryType": "DDL",
+  "Original": "truncate user.user_extra",
+  "Instructions": {
+    "OperatorType": "DDL",
+    "Keyspace": {
+      "Name": "user",
+      "Sharded": true
+    },
+    "Query": "truncate table user_extra"
+  }
+}
+
+# Rename table
+"rename table a to main.b"
+{
+  "QueryType": "DDL",
+  "Original": "rename table a to main.b",
+  "Instructions": {
+    "OperatorType": "DDL",
+    "Keyspace": {
+      "Name": "main",
+      "Sharded": false
+    },
+    "Query": "rename table a to b"
+  }
+}
+
+# Flush statement
+"flush tables unsharded, music"
+{
+  "QueryType": "FLUSH",
+  "Original": "flush tables unsharded, music",
+  "Instructions": {
+    "OperatorType": "Send",
+    "Keyspace": {
+      "Name": "main",
+      "Sharded": false
+    },
+    "TargetDestination": "AllShards()",
+    "IsDML": false,
+    "Query": "flush tables unsharded, music",
+    "SingleShardOnly": false
+  }
+}
```

Comment:
this plan does not look correct. `music` table is in `user` keyspace and `unsharded` is in `main` keyspace. So it should have a `concatenate` primitive on top.

Reply:
this is correct, since the default keyspace is set, tables are searched only in this keyspace.

Reply:
The default is set to unsharded keyspace that's is why it does not error out. got it.